src/proto/FunctionRpc.proto

```diff
@@ -65,6 +65,10 @@ message StreamingMessage {
 
     // Worker logs a message back to the host
     RpcLog rpc_log = 2;
+
+    FunctionEnvironmentLoadRequest function_environment_load_request = 25;
```

Comment:
Can we call it something like "FunctionEnvironmentReloadRequest", since environment variables will be there by default for most cases?

Reply:
makes sense. will update